src/pages/benefits/viewAllBenefit/ApplicationLists.tsx

```
@@ -51,24 +41,25 @@ const DetailsButton = ({ rowData }: ICellTextProps) => {
   const [pageIndex, setPageIndex] = useState(0);
   const pageSize = 10;
+
   useEffect(() => {
     const fetchApplicationData = async () => {
       if (id) {
         try {
-          const applicantionDataResponse = await viewAllApplicationByBenefitId(
-            id
-          );
+          const applicantionDataResponse = await viewAllApplicationByBenefitId(id);
           const processedData = applicantionDataResponse?.map((item: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)